domain-services/ifix-department-entity-service/README.md

```diff
@@ -27,5 +27,20 @@ Children list contains department entity id list, which makes current department
 1. First we need to define the hierarchy level top to bottom because it has parent's reference.
 2. Only then we can start adding department entities, bottom-to-top because it has child's reference.
 
-_**Note**: The root department hierarchy will have the label "Department", and the root department entity will be the
+_**Note**: 
+1. The root department hierarchy will have the label "Department", and the root department entity will be the
 department itself._
+2. When we have to update the existing children list of Department Entity then update the existing children list using mongodb command like below:
+
+   **Steps**: 
+             
+             1. Find the department entity which hierarchy level is less than one by current department entity's hierarchy level.
+              
+              db.departmentEntity.find({"departmentId" : "<current_department_entity's_department_id>","hierarchyLevel": <Current_department_entity_hierarchy_level- 1>});
```

Comment:
Write: we need to find the department entity parent where the new children need to be added. Do search by name and hierarchy level, and not just hierarchy level.

Reply:
done